apps/demo-e2e/src/integration/app.spec.ts

```diff
@@ -158,4 +157,11 @@ describe('demo', () => {
     cy.get('.submit-payment-btn').click()
     cy.get('[cy-data="panel-submit-payment"]')
   })
+
+  it('should submit a payment batch', () => {
+    cy.seedDb()
+    cy.get('[cy-data="cy-nav-btn-sdk"]').click()
+    cy.get('.submit-batch-payments-btn').click()
+    cy.get('[cy-data="panel-submit-payment-batch"]')
+  })
```

Comment:
I think it's missing an `expect` here? 

I think we want to see that it returned something and at least the `source` is what we expect it to be.